core/sodasql/scan/scan_builder.py

```diff
@@ -69,7 +72,15 @@ def __init__(self):
         self.assert_no_warnings_or_errors = True
         self.soda_server_client: SodaServerClient = None
 
+    def __call__(self, *args, **kwargs):
```

Comment:
I don't see how `__call__` helps, it is better to add the version check to `__init__`

Reply:
Update done. Thanks